sdk/test-utils/perf/src/managerProgram.ts

```diff
@@ -0,0 +1,300 @@
+import { exec } from "child_process";
+import os from "os";
+import { performStage } from "./barrier";
+import {
+  ReportResultsMessage,
+  StatusUpdateMessage,
+  WorkerToManagerMessageWithId,
+} from "./messages";
+import { ManagerMulticoreUtils, multicoreUtils } from "./multicore";
+import { DefaultPerfOptions, ParsedPerfOptions } from "./options";
+import { Snapshot } from "./snapshot";
+import { PerfTestBase, PerfTestConstructor } from "./perfTestBase";
+import { PerfProgram } from "./program";
+import { formatDuration } from "./utils";
+
+/**
+ * PerfProgram
+ * receives a class extending PerfTest with specific command line parameter names (or just "string"),
+ * then gets the test ready for a performance/stress test run.
+ *
+ * Use it like:
+ *
+ * ```ts
+ * export class Delay500ms extends PerfTest {
+ *   public options = {};
+ *   async run(): Promise<void> {
+ *     await delay(500);
+ *   }
+ * }
+ *
+ * const perfProgram = new PerfProgram(Delay500ms);
```

Comment:
not applicable anymore?

Reply:
Yeah, this whole doc doesn't belong here. I'll put it with `createPerfProgram` which is the new public API.